staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -81,9 +81,6 @@ if ($global:NetworkPlugin -eq "azure") {
     # Restart Kubeproxy, which would wait, until the network is created
     # This was fixed in 1.15, workaround still needed for 1.14 https://github.com/kubernetes/kubernetes/pull/78612
     Restart-Service Kubeproxy -Force
-
-    # Set env file for Azure Stack
-    $env:AZURE_ENVIRONMENT_FILEPATH = "c:\k\azurestackcloud.json"
```

Comment:
If this is removed, kubelet will not be able to get endpoints for custom clouds.

Reply:
reverted this change

Reply:
@cameronmeissner If you plan to support custom clouds with other networkplugin, maybe you need to move `$env:AZURE_ENVIRONMENT_FILEPATH = "c:\k\azurestackcloud.json"` to outside of if.

Reply:
@cameronmeissner I think that you can remove it if a persistent environment variable `AZURE_ENVIRONMENT_FILEPATH` exists

Reply:
> @cameronmeissner I think that you can remove it if a persistent environment variable `AZURE_ENVIRONMENT_FILEPATH` exists

right, this is what I was thinking - I figured we could just always set it irregardless of network plugin, that way whenever a particular plugin needs it it'll just be there, as well as for the secure bootstrap client